src/ansys/dpf/core/operators/scoping/adapt_with_scopings_container.py

```diff
@@ -118,6 +118,12 @@ def _spec() -> Specification:
                     optional=True,
                     document=r"""Whether to keep fields that become empty after rescoping. Default is false.""",
                 ),
+                2: PinSpecification(
```

Comment:
@ansys-akarcher what do the changes in this operator and in the following one have to do with the objective of the PR?